docs/topics/create-basic-yaml-rule-template.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="create-basic-yaml-rule-template_{context}"]
+= Creating a basic YAML rule template
+
+{ProductShortName} YAML-based rules have the following basic structure:
+
+[source,terminal]
+----
+when(condition)
+ message(message)
+ tag(tags)
+----
+.Procedure
+
+. In the `/home/<USER>/` directory, create a file containing the basic syntax for YAML rules as follows:
++
+[source,terminal]
+----
+- category: mandatory
+  description: |
+   <DESCRIPTION TITLE>
+   <DECRIPTION TEXT>
```

Comment:
DESCRIPTION

Reply:
After GA, see if the labels/parameters should be all caps; usually we write lower-case.